sdk/redisenterprise/Azure.ResourceManager.RedisEnterprise/api/Azure.ResourceManager.RedisEnterprise.netstandard2.0.cs

```diff
@@ -521,4 +599,11 @@ public partial class RedisPersistenceSettings
         public bool? IsRdbEnabled { get { throw null; } set { } }
         public Azure.ResourceManager.RedisEnterprise.Models.PersistenceSettingRdbFrequency? RdbFrequency { get { throw null; } set { } }
     }
+    public partial class RegionSkuDetail
```

Comment:
Please rename this class to `RedisEnterpriseRegionSkuDetail`

Reply:
Updated